MapleServer2/Servers/Game/GameSession.cs

```diff
@@ -31,23 +31,6 @@ public class GameSession : Session
         public GameSession(ManagerFactory<FieldManager> fieldManagerFactory, ILogger<GameSession> logger) : base(logger)
```

Comment:
Remove the declaration of `CancellationToken` too!

Reply:
done!